apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/status_test.exs

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.StatusTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+
+  @moduletag :integration
+  @moduletag :watcher
+  # a test in OMG.Watcher.Integration.StatusTest partially tests the controller, but it needs whole system setup
+  # TODO - add clean controller tests
```

Comment:
`status` controller is pure integration. We can keep such empty test here with the remark that the existing test tests the controller enough and have no `TODO`, at least as long as the controller contains no logic of its own.